src/clarity-angular/utils/_theme.dark.clarity.scss

```diff
@@ -0,0 +1,694 @@
+// END LOGIN
+
+/**********
+ * Modal
+ * Modals provide information or help a user complete a task.
+ *
+ * Usage: ../modal/_modal.clarity.scss
+ */
+$clr-modal-close-color: #adbbc4;
+$clr-modal-bg-color: #22343c;
+// END Modal
+
+/***************
+ * Nav
+ * A sound navigation layout offers a high degree of discoverability and feedback, letting users know where they are at
+ * all times and ensuring they can easily get to where they want to go.
+ *
+ * The variables in this section control the following navigation components
+ * - Responsive nav
+ * - Side nav
+ * - Sub nav
+ *
+ * Usage:
+ * - ../layout/nav/_responsive-nav.clarity.scss
+ * - ../layout/nav/_sidenav-nav.clarity.scss
+ * - ../layout/nav/_subnav.clarity.scss
+ */
+$clr-sliding-panel-width: 15rem;
+$clr-sliding-panel-width-sm: 12rem;
+$clr-sliding-panel-text-color: clr-getTextColor();
+$clr-trigger-position: $clr-sliding-panel-width + 0.5rem;
+$clr-trigger-position-sm: $clr-sliding-panel-width-sm + 0.5rem;
+$clr-sliding-panel-z-index: map-get($clr-layers, sidepanel);
+$clr-sliding-panel-backdrop-z-index: map-get($clr-layers, sidepanel-bg);
+$clr-sliding-transition-time: 0.3s;
+$clr-transition-style: ease;
+$clr-nav-background-color: $clr-light-gray;
+$clr-nav-background-color: #1b2a32;
+$clr-responsive-nav-hover-bg: $clr-global-selection-color;
+$clr-sliding-panel-text-color: $clr-white;
+
+$clr-sidenav-border-color: #152127;
+$clr-sidenav-link-hover-color: $clr-global-selection-color;
+
+$clr-subnav-bgColor: #17242b;
+$clr-nav-shadow: 0 -1px 0 $gray-light-midtone inset;
+// END Nav
+
+/**************
+ * Progress Bars
+ * A progress bar is a linear indicator for providing feedback about an ongoing, user-initiated process.
+ *
+ * Usage: ../progress/progress-bars/_progress-bars./clarity.scss
+ */
+$clr-progress-defaultBarColor: #49afd9;
+$clr-progress-success-color: #205522; // TODO confirm these with UX - reused existing success/danger colors from elsewhere
```

Comment:
Marking this as something to ensure is ok?

Reply:
I did confirm once and will remove the comment.